src/main/java/main/internet/shop/dao/jdbcimpl/UserDaoJdbcImpl.java

```diff
@@ -164,14 +167,15 @@ private User extractUserFromResultSet(ResultSet resultSet) throws SQLException {
         user.setLogin(resultSet.getString("users.login"));
         user.setName(resultSet.getString("users.name"));
         user.setPassword(resultSet.getString("users.password"));
+        user.setSalt(resultSet.getBytes("users.salt"));
```

Comment:
```suggestion
        user.setSalt(resultSet.getBytes("salt"));
```
same for all column values that you get from result set